src/integration/test_suites/smoke_test_sha3_interrupt/smoke_test_sha3_interrupt.c

```diff
@@ -63,32 +66,38 @@ void main() {
     //////////////////////////////////
     // Enable FIFO empty interrupt. //
     //////////////////////////////////
+    // Currently we cannot fill the FIFO using just 32-bit writes, so test the empty interrupt using the interrupt test register.
```

Comment:
I'd suggest adding a note here that explains we're actually timing the interrupt test register, and that's why the timeout shouldn't depend on the number of entries in the fifo.

Reply:
I'm not sure I know what you mean here by "timing the interrupt test register". The main purpose of this test is to make sure each of the interrupts can be fired and received by software. I'm not really timing anything.